### PR TITLE
Change module name and update dependencies

### PR DESCRIPTION
**Pull request by brecode**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: 2.1
 
 # Docker environment.
 defaults: &defaults
-  working_directory: /go/src/github.com/cisco-app-networking/nsm-nse
```
Comment (adodon2go) [inline .circleci/config.yml:5]: I don't think we need to update this path here and below in this file. This file can stay unchanged.

File: api/ipam/ipprovider/ipprovider.pb.go [hunk 2]
```diff
@@ -1,28 +1,33 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
```
Comment (adodon2go) [inline api/ipam/ipprovider/ipprovider.pb.go:2]: This seams to be an unrelated change. Any particular reason why we want to push this changes in this PR?

File: api/ipam/ipreceiver/ipreceiver.pb.go [hunk 3]
```diff
@@ -1,160 +1,269 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
```
Comment (adodon2go) [inline api/ipam/ipreceiver/ipreceiver.pb.go:2]: Same observation as above

File: build/ci/runner/Dockerfile.runner [hunk 4]
```diff
@@ -1,19 +1,19 @@
 ARG BASEIMG=ciscoappnetworking/kubectl_kind:kubectl.v1.17.5-kind.v0.7.0-helm.v3.3.4
 FROM ${BASEIMG}
 
-ARG  vl3_repo=https://github.com/cisco-app-networking/nsm-nse.git
+ARG  vl3_repo=https://cisco-app-networking.github.io/nsm-nse.git
```
Comment (adodon2go) [inline build/ci/runner/Dockerfile.runner]: Don't change this. This repo doesn't exist.

File: build/ci/runner/Dockerfile.runner [hunk 5]
```diff
@@ -1,19 +1,19 @@
...
 # set this to branch or $CIRCLE_SHA1
 ARG  vl3_branch=master
-ARG  nsm_repo=https://github.com/cisco-app-networking/networkservicemesh.git
+ARG  nsm_repo=https://cisco-app-networking.github.io/nsm-nse/networkservicemesh.git
```
Comment (adodon2go) [inline build/ci/runner/Dockerfile.runner]: This repo doesn't exist. In fact I don't think we really need to change this file.

File: build/ci/runner/run_vl3.sh [hunk 6]
```diff
@@ -33,7 +33,7 @@ for kconf in ${kubeconfs}; do
         KCONF2=${KUBECONFDIR}/${kconf}
         echo "Cluster 2 is ${KUBECONFDIR}/${kconf}"
     fi
-    pushd /go/src/github.com/cisco-app-networking/nsm-nse
+    pushd /go/src/cisco-app-networking.github.io/nsm-nse
```
Comment (adodon2go) [inline build/ci/runner/run_vl3.sh]: lets keep the old path

File: build/nse/ucnf-kiknos/Dockerfile.runner [hunk 7]
```diff
@@ -5,8 +5,8 @@ ARG  nsm_repo=https://github.com/networkservicemesh/networkservicemesh.git
 RUN  git clone $nsm_repo /go/src/github.com/networkservicemesh/networkservicemesh
 
 # Add nsm-nse code contents
-RUN mkdir -p /go/src/github.com/cisco-app-networking/nsm-nse
-COPY . /go/src/github.com/cisco-app-networking/nsm-nse
+RUN mkdir -p /go/src/cisco-app-networking.github.io/nsm-nse
```
Comment (adodon2go) [inline build/nse/ucnf-kiknos/Dockerfile.runner]: keep the old path

File: build/nse/vl3-nse/README.md [hunk 8]
```diff
@@ -29,12 +29,12 @@ This is an NSE that creates a L3 routing domain between NSC workloads. Each NSE
 1. This currently only works with a custom version of the NSM installation.
 
     ```sh
-    mkdir -p $GOPATH/src/github.com/cisco-app-networking
-    git clone https://github.com/cisco-app-networking/networkservicemesh
+    mkdir -p $GOPATH/src/cisco-app-networking.github.io/nsm-nse
```
Comment (adodon2go) [inline build/nse/vl3-nse/README.md]: don't update paths in this file and related ones

File: .circleci/config.yml (hunk 1 above)
Comment (brecode) [reply .circleci/config.yml:5]: At this point any change on circle CI is additional pain, this will go into the develop_jenkins branch and so no circle CI changes will be tested

File: api/ipam/ipprovider/ipprovider.pb.go (hunk 2 above)
Comment (brecode) [reply api/ipam/ipprovider/ipprovider.pb.go:2]: I went and generated the proto files again. Probably have an updated version of protoc. fundamentally no change

File: build/ci/runner/Dockerfile.runner (hunk 4 above)
Comment (brecode) [reply build/ci/runner/Dockerfile.runner]: check

File: build/ci/runner/Dockerfile.runner (hunk 5 above)
Comment (brecode) [reply build/ci/runner/Dockerfile.runner]: check

File: build/nse/vl3-nse/README.md (hunk 8 above)
Comment (brecode) [reply build/nse/vl3-nse/README.md]: check

**Comment by tiswanso** [comment]
closing this as it was superseded by work on the wwwin-github devop_jenkins branch
